### PR TITLE
feature/WAL-521

### PR DESCRIPTION
**Pull request by sksadjad**

### CODE REVIEW

File: src/components/containers/SSIButtonsContainer/index.tsx [hunk 1]
```diff
@@ -41,11 +42,11 @@ class SSIButtonsContainer extends PureComponent<Props, IState> {
   };
 
   render() {
-    const {primaryButton, secondaryButton} = this.props;
+    const {primaryButton, secondaryButton, style} = this.props;
     const {keyboardVisible} = this.state;
 
     return (
-      <ButtonContainer style={{marginBottom: keyboardVisible ? 18 : 36}}>
+        <ButtonContainer style={{marginBottom: keyboardVisible ? 18 : 36, ...(style && style)}}>
```
Comment (BtencateSphereon) [inline src/components/containers/SSIButtonsContainer/index.tsx:49]: style={{...style, marginBottom: keyboardVisible ? 18 : 36}}

you can just deconstruct in this case

File: src/components/views/SSITabView/index.tsx [hunk 2]
```diff
@@ -7,17 +8,18 @@ import {ITabRoute, ITabViewRoute} from '../../../types';
...
 export interface IProps {
+  style?: ViewStyle;
   routes: Array<ITabViewRoute>;
 }
 
 const SSITabView: FC<IProps> = (props: IProps): JSX.Element => {
   const [index, setIndex] = React.useState(0);
   const [routes] = React.useState(props.routes);
-
+  const style = props.style;
```
Comment (BtencateSphereon) [inline src/components/views/SSITabView/index.tsx:18]: i rather deconstruct props where possible:

const { style } = props

then you can use style where you want. you can then also try to at routes to it, but it shares the name of the state

File: src/components/views/SSITabView/index.tsx [hunk 3]
```diff
@@ -7,17 +8,18 @@ import {ITabRoute, ITabViewRoute} from '../../../types';
...
   const sceneMap = props.routes.reduce((a: Record<string, any>, v: ITabViewRoute) => ({...a, [v.key]: v.content}), {});
 
   return (
-    <Container>
+    <Container style={{...(style && style)}}>
```
Comment (BtencateSphereon) [inline src/components/views/SSITabView/index.tsx:22]: is this needed? the tabview just hold some routes, these routes should determine which color of background they need

File: src/screens/SSICredentialDetailsScreen/index.tsx [hunk 4]
```diff
@@ -62,7 +63,7 @@ const SSICredentialDetailsScreen: FC<Props> = (props: Props): JSX.Element => {
         <SSITabView routes={routes} />
         {/* TODO we use this 2 button structure a lot, we should make a component out of it */}
         {(primaryAction || secondaryAction) && (
-          <ButtonContainer>
+          <ButtonContainer style={{ backgroundColor: backgrounds.secondaryDark }}>
```
Comment (BtencateSphereon) [inline src/screens/SSICredentialDetailsScreen/index.tsx:66]: we could replace this whole view with the SSIButtonContainer component. if that is to much work, move the styling here to the correct styling component

File: src/screens/SSISummaryScreen/index.tsx [hunk 5]
```diff
@@ -65,8 +66,8 @@ class SSIOnboardingSummaryScreen extends PureComponent<IProps> {
     ];
 
     return (
-      <Container>
-        <SSITabView routes={routes} />
+      <Container style={{ backgroundColor: backgrounds.secondaryDark }}>
```
Comment (BtencateSphereon) [inline src/screens/SSISummaryScreen/index.tsx:69]: same comment here, what is the reason for adding this? and can the routes not decide which color they want? same for the styling below this

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

File: src/components/views/SSITabView/index.tsx (hunk 3 above)
Comment (sksadjad) [reply src/components/views/SSITabView/index.tsx]: removed this styling

File: src/components/views/SSITabView/index.tsx (hunk 2 above)
Comment (sksadjad) [reply src/components/views/SSITabView/index.tsx]: remvoed this styling

File: src/components/containers/SSIButtonsContainer/index.tsx (hunk 1 above)
Comment (sksadjad) [reply src/components/containers/SSIButtonsContainer/index.tsx]: removed this styling

File: src/screens/SSISummaryScreen/index.tsx [hunk 6]
```diff
@@ -65,8 +66,8 @@ class SSIOnboardingSummaryScreen extends PureComponent<IProps> {
     ];
 
     return (
-      <Container>
-        <SSITabView routes={routes} />
+      <Container style={{backgroundColor:backgrounds.secondaryDark}}>
```
Comment (BtencateSphereon) [inline src/screens/SSISummaryScreen/index.tsx:69]: Create a new styling component for this container specifically for this screen with the background color

Comment (sksadjad) [reply src/screens/SSISummaryScreen/index.tsx]: created a new component for `SSISummaryScreen` container

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**